DataConnectors/AtlassianConfluenceAudit/ConfluenceAudit_API_FunctionApp.json

```diff
@@ -0,0 +1,121 @@
+			}
+		]
+	},
+	"instructionSteps": [{
+			"title": "",
+			"description": ">**NOTE:** This connector uses Azure Functions to connect to the Confluence REST API to pull its logs into Azure Sentinel. This might result in additional data ingestion costs. Check the [Azure Functions pricing page](https://azure.microsoft.com/pricing/details/functions/) for details."
+		},
+		{
+			"title": "",
+			"description": ">**(Optional Step)** Securely store workspace and API authorization key(s) or token(s) in Azure Key Vault. Azure Key Vault provides a secure mechanism to store and retrieve key values. [Follow these instructions](https://docs.microsoft.com/azure/app-service/app-service-key-vault-references) to use Azure Key Vault with an Azure Function App."
+		},
+		{
+			"description": ">**NOTE:** This data connector depends on a parser based on a Kusto Function to work as expected. [Follow these steps](https://aka.ms/sentinel-confluenceauditapi-parser) to create the Kusto functions alias, **ConfluenceAudit**"
+		},
+		{
+			"title": "",
+			"description": "**STEP 1 - Configuration steps for the Confluence API**\n\n [Follow the instructions](https://developer.atlassian.com/cloud/confluence/rest/intro/#auth) to obtain the credentials. \n"
+		},
+		{
+			"title": "",
+			"description": "**STEP 2 - Choose ONE from the following two deployment options to deploy the connector and the associated Azure Function**\n\n>**IMPORTANT:** Before deploying the Workspace data connector, have the Workspace ID and Workspace Primary Key (can be copied from the following).",
+			"instructions": [{
+					"parameters": {
+						"fillWith": [
+							"WorkspaceId"
+						],
+						"label": "Workspace ID"
+					},
+					"type": "CopyableLabel"
+				},
+				{
+					"parameters": {
+						"fillWith": [
+							"PrimaryKey"
+						],
+						"label": "Primary Key"
+					},
+					"type": "CopyableLabel"
+				}
+			]
+		},
+		{
+			"title": "Option 1 - Azure Resource Manager (ARM) Template",
+			"description": "Use this method for automated deployment of the Confluence Audit data connector using an ARM Tempate.\n\n1. Click the **Deploy to Azure** button below. \n\n\t[![Deploy To Azure](https://aka.ms/deploytoazurebutton)](https://aka.ms/sentinelconfluenceauditazuredeploy)\n2. Select the preferred **Subscription**, **Resource Group** and **Location**. \n> **NOTE:** Within the same resource group, you can't mix Windows and Linux apps in the same region. Select existing resource group without Windows apps in it or create new resource group.\n3. Enter the **ConfluenceAccessToken**, **ConfluenceUsername**, **ConfluenceHomeSiteName** (short site name part, as example HOMESITENAME from https://HOMESITENAME.atlassian.net) and deploy. \n4. Mark the checkbox labeled **I agree to the terms and conditions stated above**. \n5. Click **Purchase** to deploy."
```

Comment:
please update short url link for azure deploy as follows:
https://aka.ms/sentinel-confluenceauditapi-azuredeploy

Reply:
fixed